Views/Home/Index.cshtml

```diff
@@ -1,5 +1,24 @@
 <div class='main'>
     <img class='phone-img' src='~/images/phone.png' alt='Phone'/>
-    <div class='separator'></div>
+    <div class='separator-y'></div>
     <img class='logo-img' src='~/images/tcg_logo.png' alt='Logo'/>
 </div>
+
+<div class='separator-x'></div>
```

Comment:
although this works, if you use a `::before` pseudoelement  in the features selector you can have this purely aesthetic element done using just css. set the `features` div to position relative and the before pseudoelement to position absolute to place the before pseudoelement manually with a position relative to the features element. If this sounds complicated contact me and we can check it.